### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by chriswmercer**
#538 Add content menu to CONTRIBUTING
Please note, the ticket says to add before Installation, but this md doesn't have an installation section, so i've added it at a place that feels appropriate.

This is for issue 538 which was to add contents to CONTRIBUTING.md. Please note, the ticket says to add before Installation, but this md doesn't have an installation section, so i've added it at a place that feels appropriate.

Issue: 538

### CODE REVIEW

**Comment by shahednasser** [comment]
Oh sorry I meant before Instructions. Can you change it to that

**Comment by chriswmercer** [comment]
Oh yes sure no probs

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,17 @@ Please note we have a code of conduct, please follow it in all your interactions
 2. Before contributing to an issue, make sure that no one is assigned or has taken that issue. If no one is and you would like to work on the issue, please comment on the issue to let others know that someone is working on the issue.
 3. Before creating a pull request, it is **important** that you make sure your branch and repository is up to date with this one. Some conflicts can be resolved, but many are hard to resolve. **Add the upstream** branch and always keep your branch up to date.
 
+- [Instructions](#instructions)
```
Comment (shahednasser) [inline CONTRIBUTING.md:15]: Please add the title "Content" to it

Comment (chriswmercer) [reply CONTRIBUTING.md:15]: Already did it sorry, I didn't like how it looked :)

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @chriswmercer for doc

**Comment by chriswmercer** [comment]
Thank you :)

**Pull request merged**